.travis.yml

```diff
@@ -24,20 +24,27 @@ env:
   - DB=sqlite
   - DB=mysql
   - DB=pgsql
+  - DB=memcache
   - DB=memcached
 
 install:
+  # install required packages
   - travis_retry composer self-update
-  - travis_retry composer install --dev --no-interaction --prefer-source
+  - travis_retry composer install -o --no-interaction --prefer-source
+
+  # create log dir
   - mkdir -p build/logs
   - tests/bin/travis-init.sh
 
 script:
-  - vendor/bin/phpunit --colors --verbose --coverage-text --coverage-clover build/logs/clover.xml
+# xdebug is not installed on HHVM and PHP 7, so we are skipping the creation of code coverage data there
```

Comment:
These changes should go in there own PR. They are not directly related to your `memcache` integration.